docs/glossary.md

```diff
@@ -280,7 +280,14 @@ mrs-developer
     As a byproduct of its update operations, it also automatically adjusts `jsconfig.json`, which is used by Volto to configure webpack aliases.
 
 Yarn
-    [Yarn](https://yarnpkg.com/) is a JavaScript package manager.
+    [Yarn](https://yarnpkg.com/) is both a JavaScript package manager and project manager.
+
+Corepack
+    [Corepack](https://github.com/nodejs/corepack) is a zero-runtime-dependency Node.js script that acts as a bridge between Node.js projects and the package managers they are intended to be used with during development.
+    In practical terms, Corepack lets you use {term}`Yarn`, {term}`npm`, and {term}`pnpm` without having to install them.
```

Comment:
Do we have an `pnpm` term?

Reply:
Yup, near the end of the Glossary, which automatically sorts on rendering to HTML.

```markdown
pnpm
    [pnpm](https://pnpm.io/) is a fast, disk space efficient package manager.
```